src/shared/components/ui/popover/PopoverContent.tsx

```diff
@@ -0,0 +1,241 @@
+'use client';
+import { ReactNode, useEffect, useLayoutEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+import { cn } from '@/shared/libs/cn';
+
+import { usePopover } from './usePopover';
+
+/**
+ * Popover 컨텐츠의 위치를 정의하는 타입
+ *
+ * @description 트리거 요소를 기준으로 12가지 방향으로 배치 가능
+ * - bottom: 아래쪽 (start: 왼쪽 정렬, center: 중앙 정렬, end: 오른쪽 정렬)
+ * - top: 위쪽 (start: 왼쪽 정렬, center: 중앙 정렬, end: 오른쪽 정렬)
+ * - left: 왼쪽 (start: 위쪽 정렬, center: 중앙 정렬, end: 아래쪽 정렬)
+ * - right: 오른쪽 (start: 위쪽 정렬, center: 중앙 정렬, end: 아래쪽 정렬)
+ */
+type PopoverPosition =
+  | 'bottom-start'
+  | 'bottom-center'
+  | 'bottom-end'
+  | 'top-start'
+  | 'top-center'
+  | 'top-end'
+  | 'left-start'
+  | 'left-center'
+  | 'left-end'
+  | 'right-start'
+  | 'right-center'
+  | 'right-end';
+
+/**
+ * PopoverContent 컴포넌트의 Props 인터페이스
+ *
+ * @interface PopoverContentProps
+ * @extends {React.HTMLAttributes<HTMLDivElement>} HTML div 요소의 모든 속성을 상속
+ */
+interface PopoverContentProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
이 부분도 저희 컨벤션에 따라 `import` 해서 사용하시는 것 어떨지 권장드립니다!
```suggestion
interface PopoverContentProps extends HTMLAttributes<HTMLDivElement> {
```

Reply:
여길 놓쳤군요.. 수정해놓겠습니다!